lib/graphql_rails/attributes/attribute.rb

```diff
@@ -43,14 +44,22 @@ def property(new_property = nil)
         self
       end
 
+      def camelize(new_camelize = nil)
```

Comment:
i would suggest to use `options` instead of `camelize`. This way you will be more consistent with https://samesystem.github.io/graphql_rails/#/components/controller?id=options